app/ocr_strategies/marker.py

```diff
@@ -1,12 +1,13 @@
 from marker.convert import convert_single_pdf
+from marker.convert import convert_single_pdf
 from marker.models import load_all_models
 
 from ocr_strategies.ocr_strategy import OCRStrategy
 
 class MarkerOCRStrategy(OCRStrategy):
     """Marker OCR Strategy"""
-    def extract_text_from_pdf(self, pdf_bytes):
+    def extract_text_from_pdf(self, file_bytes):
```

Comment:
shouldn't it support FileFormats as `llama_vision` here?

Reply:
hmm in this case if someone passes `png` as an input file we should convert it to pdf first?  or just return an exception (it's fine for now) that this is not supported for `marker` strategy

Reply:
@pkarw That was one of the things left in the stash; I completely forgot where is it but I was sure I did it. :D Of course now it's merged and even fixed after tests. :)